### PR TITLE
Vehicle Image Classification

### PR DESCRIPTION
**Pull request by Soumayan-pal01**
## Vehicle Image Classification

- The aim of this project is to classify a vehicle image which type of vehicle it is.First we will try to visualize the training images to understand how to perform this multiclass image classification. Then we will use Neural Network approaches: MLP and CNN, to train the model and finally compare the validation accuracy of both the models.The dataset contain 7 classes of vehicles.

- [x] LGM-SOC'21 Participant
- [ ] DevIncept Participant 
- [ ] Contributor

Closes: #365

### Describe the add-ons or changes you've made

In this project we have used a couple of Deep Learning frameworks to recognize the type of vehicle that is given in the images. The MLP model performs fairly but gets slightly overfitted. However, CNN approach is then used to train the dataset which performs well and gives a validation accuracy of 87.12% .

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by prathimacode-hub — APPROVED** [review]
Clean and perfect PR. Impressive work. Keep rising high. This is a gem. 🤩🙌👌@Soumayan-pal01

**Comment by Soumayan-pal01** [comment]
> Clean and perfect PR. Impressive work. Keep rising high. This is a gem. 🤩🙌👌@Soumayan-pal01

Thanks 😇@prathimacode-hub
